### ISSUE TITLE
Practical 9B: Neural Network using Keras (Simple Example)

### ISSUE THREAD

**Issue by karinkato2103** [karinkato2103/soft#18]
import numpy as np
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense

if __name__ == "__main__":
    X = np.array([[0,0],[0,1],[1,0],[1,1]])
    y = np.array([0,1,1,0])

    model = Sequential()
    model.add(Dense(4, input_dim=2, activation="relu"))
    model.add(Dense(1, activation="sigmoid"))

    model.compile(loss="binary_crossentropy", optimizer="adam", metrics=["accuracy"])
    model.fit(X, y, epochs=500, verbose=0)

    print(model.predict(X))
